### PR TITLE
255/hamburger+responsive

### PR DESCRIPTION
**Pull request by arekmano**
- Added responsiveness to scroll fix
- Added chat history tweak to make it more message-like
- Added Transition to about us tab, removed transparency.

[#254, #255]

### CODE REVIEW

**Comment by arekmano** [comment]
Chat history update: (offet either to left or to the right) image

**Comment by arekmano** [comment]
This PR contains #266

File: src/beta_client/src/components/Beta.vue [hunk 1]
```diff
@@ -26,20 +26,22 @@
                 <chat :language="language"></chat>
             </div>
         </div>
-        <div id="beta-page-about-us" v-if="aboutUs">
-            <div id="beta-page-about-us-nav">
-                <p v-on:click="aboutUs=!aboutUs">CLOSE</p>
+        <transition name="translate">
```
Comment (dodoels) [inline src/beta_client/src/components/Beta.vue:29]: I thought the animation is not needed?

**Review by dodoels — APPROVED** [review]
Syntax wise no issue but please make sure responsive (340px and up in width;  700px and up in height) is working.

**Comment by arekmano** [comment]
works. can confirm

**Pull request merged**